front/app/containers/App/index.tsx

```diff
@@ -314,6 +323,7 @@ const App = ({ children }: Props) => {
           >
             <GlobalStyle />
             <Box
+              className={appContainerClassName}
```

Comment:
Is it necessary to apply this class name conditionally? Can't we just always apply it, and call it "app-container" or something like that?

And maybe we can make it an `id` instead since there will only be once instance of this component?

Reply:
> Is it necessary to apply this class name conditionally? Can't we just always apply it, and call it "app-container" or something like that?

@luucvanderzee For this ticket, we only want the weglot selector to receive a margin when the current user is an admin or project manager. So here I apply the "admin-user-view" classname when the user is an admin. I don't want to shift the weglot button if the user is not an admin - so I think I need to apply this class conditionally? :thinking: Or would you have another suggestion here maybe?

Reply:
Ah yes you're totally right. I thought I needed it in a state variable for when the user logs in/out, but I don't. Fixing it up now :) Thanks - sorry I should have thought of that!